### ISSUE TITLE
Generator without classes

### ISSUE THREAD

**Issue by aguacaate** [metal3d/keras-video-generators#27]
Hey, 
Thank you very much for your video generator!
I am now trying to train a LSTM with image sequences but as I want to train for a future GAN implementation I have no classes. How would I be able to apply your generator with no classes?
Would be grateful if you could help.

Thanks a lot
Cate

**Comment by metal3d** [comment]
I already tried (without good results for now). The idea is to make the "generator" (I mean the model part, not the VideoFrameGenerator) to output a shape of (B, N, W ,H , C) where B is the batch size (None ?) and N the number of frame.

Then use VideoFrameGenerator (or SlidingFrameGenerator) dropping class:

```python
# index 0 is the batches of sequeces, index 1 is classes, you can ignore this index
bach = next(your_generator)[0]
```

I think the idea can be very interesting without using data augmentation (or not destructive).

I will try to give more details in my Medium account (my blog is french, my Medium is english)

**Comment by aguacaate** [comment]
Yes! Actually I am trying to develop a many-to-one model so the output of the generator should only be one next frame. 
Will work on it and share if I manage it successfully!
Thanks a lot for your reply and Ill wait for your Medium post.
